tests/benchmarks/spi_endless/testcase.yaml

```diff
@@ -7,9 +7,11 @@ common:
   build_only: true
   platform_allow:
     - nrf52840dk/nrf52840
+    - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Support for PDK can be remove for this tests - no need to keep pdk files, only dk can stay.

Reply:
done